### ISSUE TITLE
frequency plot 

### ISSUE THREAD

**Issue by troddick** [USGS-WiM/NSS#42]
In the Frequency Plot, the "Disabled with Log" font is too big.  Go with a font that is smaller than the "grid" and "tics" font if you can, but consider making it bold or red to stand out when present.  Also, it's confusing why it exists for x-axis and not y-axis if both are log units.

**Comment by troddick** [comment]
For some reason (I've submitted an issue in the highcharts library's github repo), but when minor ticks and grids are active with log, the line draws outside of visible range. This only happens on the y-axis, as it draws too far up out of view. 
I can remove the minor ticks/grids for the x-axis to make it more consistent if you'd like? I left them in there because they worked on the one axis but didn't work on the other. 
I made the font size smaller and bold for the "Disabled with Log".
